assets/dev/js/frontend/utils/lightbox/lightbox.js

```diff
@@ -189,27 +293,49 @@ module.exports = elementorModules.ViewModule.extend( {
 			if ( slide.video ) {
 				$slide.attr( 'data-elementor-slideshow-video', slide.video );
 
-				var $playIcon = $( '<div>', { class: classes.playButton } ).html( $( '<i>', { class: classes.playButtonIcon } ) );
+				const $playIcon = $( '<div>', { class: classes.playButton } ).html( $( '<i>', { class: classes.playButtonIcon } ) );
 
 				$slide.append( $playIcon );
 			} else {
-				var $zoomContainer = $( '<div>', { class: 'swiper-zoom-container' } ),
-					$slideImage = $( '<img>', { class: classes.image + ' ' + classes.preventClose, src: slide.image } );
+				const $zoomContainer = $( '<div>', { class: 'swiper-zoom-container' } ),
+					$slideImage = $( '<img>', {
+						class: classes.image + ' ' + classes.preventClose,
+						src: slide.image,
+						'data-title': slide.title,
+						'data-description': slide.description,
+					} );
 
 				$zoomContainer.append( $slideImage );
-
 				$slide.append( $zoomContainer );
 			}
 
 			$slidesWrapper.append( $slide );
 		} );
 
+		this.elements.$container = $container;
+		this.elements.$header = self.getSlideshoHeader();
+		this.elements.$footer = self.getSlideshowFooter();
+
+		$container.prepend( this.elements.$header );
 		$container.append(
 			$slidesWrapper,
 			$prevButton,
-			$nextButton
+			$nextButton,
+			this.elements.$footer,
 		);
 
+		this.setSettings( 'hideUiTimeout', '' );
+
+		$container.on( 'mouseenter mouseover mousedown mousemove keyup keypress swipe tap vmousedown vmouseover vmousemove', function() {
```

Comment:
1. 'mouseover' and 'mouseenter' are included in 'mousemove' event
2. 'keyup' is included in 'keypress' event
3. 'swipe', 'tap', 'vmousedown', 'vmouseover' and 'vmousemove' events are not exists